app/src/org/commcare/views/widgets/Prototype1.java

```diff
@@ -0,0 +1,148 @@
+package org.commcare.views.widgets;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+import org.commcare.dalvik.R;
+import org.javarosa.core.model.data.DateData;
+import org.javarosa.core.model.data.IAnswerData;
+import org.javarosa.form.api.FormEntryPrompt;
+
+import java.util.Calendar;
+import java.util.Date;
+
+
+/**
+ * Created by Saumya on 6/1/2016.
+ */
+public class Prototype1 extends QuestionWidget {
+
+    private CalendarWidget myCal;
+    private GregorianDateWidget myGreg;
+    private ImageButton openCalButton;
+
+    public Prototype1(Context con, FormEntryPrompt prompt){
+        super(con, prompt);
+        myGreg = new GregorianDateWidget(con, prompt);
+        myCal = new CalendarWidget(con, prompt);
+
+        myCal.setVisibility(GONE);
+
+        myGreg.removeQuestionText();
+        myCal.removeQuestionText();
+
+        addView(myCal);
+        addView(myGreg);
+
+        initButtons();
+    }
+
+    private void initButtons() {
+        openCalButton = new ImageButton(getContext());
+
+        openCalButton.setImageResource(R.drawable.avatar_vellum_date);
+
+        RelativeLayout layout = (RelativeLayout) findViewById(R.id.widgetinfo);
+        layout.addView(openCalButton);
+        RelativeLayout.LayoutParams params = (RelativeLayout.LayoutParams) openCalButton.getLayoutParams();
+        params.addRule(RelativeLayout.RIGHT_OF, R.id.gregdayofweek);
+        params.width = 60;

```

Comment:
Slightly repeating above comments, but usually we wouldn't hard code but would have things be proportional to the screen size